pandas/tests/indexes/common.py

```diff
@@ -34,6 +34,12 @@ def verify_pickle(self, index):
         unpickled = tm.round_trip_pickle(index)
         assert index.equals(unpickled)
 
+    def _fix_tz(self, new_index, orig_index):
+        if hasattr(orig_index, 'tz'):
```

Comment:
really don't spell things like this. rather change ``create_index()`` to work properly.